build/deploy.ts

```diff
@@ -43,6 +44,8 @@ async function commitIfUpdated(scriptName: string): Promise<DeployedScript | und
 }
 
 async function commitUpdate(scriptName: string, version: string): Promise<DeployedScript> {
+    // Update the changelog
+    await updateChangelog(scriptName, version, distRepo, prInfo);
```

Comment:
Do I understand this correctly, that the current code would only update the changelog for new PRs?
If that is the case, should we manually run this for already merged PRs and/or for all already existing commits in the `dist` branch?

Reply:
Yes, it'll only generate one entry for the current deployment, assuming the previous changelog was complete. Otherwise, generating a full changelog would require traversing the entire git history of the dist branch. I actually wasn't going to generate a changelog retroactively but it's probably a good idea to do that. I'll write up a quick script to generate the changelog for ECAU in the `dist` branch. Any changes prior to the dist branch should probably be excluded though, especially the ones where there were no conventional commits.

Reply:
https://github.com/ROpdebee/mb-userscripts/blob/dist/mb_enhanced_cover_art_uploads.changelog.md